lib/engine/game/g_1867/step/loan_operations.rb

```diff
@@ -23,17 +23,17 @@ def skip!
             entity = current_entity
 
             owed = @game.pay_interest!(entity)
-            if owed
-              @log << "#{entity.name} owes #{@game.format_currency(owed)} in loan "\
-                      "interest but only has #{@game.format_currency(entity.cash)}"
-              @game.nationalize!(entity)
-              # @todo: will this skip the rest of the entities turn?
-              return
-            end
+            return unpaid_interest(entity, owed) if owed
 
             @game.repay_loan(entity, entity.loans.first) while can_payoff?(entity)
             @game.calculate_corporation_interest(entity)
           end
+
+          def unpaid_interest(entity, owed)
```

Comment:
This is better, but I've changed my mind about the name of the method I suggested. `unpaid_interest` sounds like it should calculate the amount of interest that is unpaid. `unpaid_interest!` or `interest_unpaid!` would be better as they suggest it is an action that occurs when the interest is unpaid (or `credit_default!`, or `interest_default!`, or anything else you can think of.

Reply:
fixed